### PR TITLE
Adding Form 0966 intent to file documentation

### PR DESCRIPTION
**Pull request by bastosmichael**
## Description of change
Applies to ticket https://github.com/department-of-veterans-affairs/vets-contrib/issues/1917

Creating docs for ITF implementation, also cleaning up anything missed from the last PR.

## Testing done
- Testing against a local version of the developer portal

## Testing planned
- Making sure the developer portal holds the changes well.

## Acceptance Criteria (Definition of Done)
- Get a Proper Intent to File Controller documentation up on Claims V1 and V0

#### Unique to this PR
- [ ] Controller will have ITF documentation

#### Applies to all PRs

- [x] Appropriate logging
- [x] Swagger docs have been updated, if applicable
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [ ] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

File: modules/claims_api/app/swagger/claims_api/form_0966_v0_controller_swagger.rb [hunk 1]
```diff
@@ -3,5 +3,235 @@
...
+      operation :post do
+        key :summary, 'Accepts 0966 Intent to File form submission'
+        key :description, 'Accepts JSON payload. Full URL, including\nquery parameters.'
+        key :operationId, 'post0966itf'
+        key :tags, [
+          'Intent to File'
+        ]
+
+        parameter do
+          key :name, 'api_key'
```
Comment (dvogel) [inline modules/claims_api/app/swagger/claims_api/form_0966_v0_controller_swagger.rb]: This should be `apikey` (no underscore) unless something about swagger blocks transforms it.

File: modules/claims_api/app/swagger/claims_api/form_0966_v1_controller_swagger.rb [hunk 2]
```diff
@@ -3,5 +3,235 @@
...
+      operation :post do
+        key :summary, 'Accepts 0966 Intent to File form submission'
+        key :description, 'Accepts JSON payload. Full URL, including\nquery parameters.'
+        key :operationId, 'post0966itf'
+        key :tags, [
+          'Intent to File'
+        ]
+
+        parameter do
+          key :name, 'bearer_token'
```
Comment (dvogel) [inline modules/claims_api/app/swagger/claims_api/form_0966_v1_controller_swagger.rb]: Is this accurate? Should it not be `apikey` too?

Comment (bastosmichael) [reply modules/claims_api/app/swagger/claims_api/form_0966_v1_controller_swagger.rb]: V1 is bearer token based and opened like veteran verification, v0 is apikey based.

File: modules/claims_api/app/swagger/claims_api/form_0966_v0_controller_swagger.rb (hunk 1 above)
Comment (bastosmichael) [reply modules/claims_api/app/swagger/claims_api/form_0966_v0_controller_swagger.rb]: Thanks for the heads up, I'll correct this

**Pull request merged**